Core/src/org/sleuthkit/autopsy/texttranslation/translators/BingTranslatorSettings.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Autopsy
+ *
+ * Copyright 2019 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.autopsy.texttranslation.translators;
+
+import org.apache.commons.lang3.StringUtils;
+import org.sleuthkit.autopsy.coreutils.ModuleSettings;
+
+/**
+ * Class to handle the settings associated with the GoogleTranslator
```

Comment:
The comments in this file mistakenly refer to GoogleTranslator

Reply:
oops, will fix.